include/crm/common/xml.h

```diff
@@ -77,6 +77,16 @@ void fix_plus_plus_recursive(xmlNode * target);
  */
 xmlNode *create_xml_node(xmlNode * parent, const char *name);
 
+/*
+ * Create a node named "name" as a child of "parent", giving it the provided
+ * text content.
+ * If parent is NULL, creates an unconnected node.
+ *
+ * Returns the created node
+ *
+ */
+xmlNode *pcmk__create_xml_text_node(xmlNode * parent, const char *name, const char *content);
```

Comment:
I'm not sure why most XML functions were made public API to begin with, but since this is parallel to create_xml_node() I think it would be good to follow that, so this should be named with a single underbar. If we want it to be private instead, we have to add \internal to the comment, as everything in this file without that will go into the public API documentation.